Core/CoreTiming.cpp

```diff
@@ -240,6 +240,11 @@ u64 GetTicks()
 	return (u64)globalTimer + slicelength - currentMIPS->downcount;
 }
 
+u64 GetTicksPerSecond()
```

Comment:
I think we prefer using `usToCycles` and such to this... can you remove?

-[Unknown]